docs/cloud/how-to-manage-audit-logging.md

```diff
@@ -0,0 +1,205 @@
+  - Configure observability: `UpdateAccount`
+- User
+  - Create user invitations: `InviteUsers`
+  - Delete users: `DeleteUser`
+  - Update user account-level Roles: `UpdateUser`
+  - Update user Namespace permissions: `UpdateUserNamespacePermissions`
+  - Log in user: `UserLogin`
+- Namespace
+  - Create Namespace: `CreateNamespace`
+  - Update Namespace: `UpdateNamespace`
+  - Delete Namespace: `DeleteNamespace`
+  - Add or update certificates or certificate filters: `UpdateNamespace`
+  - Add custom Search Attributes: `UpdateNamespace`
+  - Rename custom Search Attribute: `RenameCustomSearchAttribute`
+  - Request increase in Retention Period: `UpdateNamespace`
+
+### Audit Log format
+
+The log sent to the Kinesis stream is JSON in the following format:
+
+```json
+{
+  "emit_time": // Time the operation was recorded
+  "level": // Level of the log entry, such as info, warning, or error
+  "user_email":  // Email address of the user who initiated the operation
+  "operation":  // Operation that was performed
+  "details":  // Details of the operation
+  "status": // Status, such as OK or error
+  "category":  // Category of the log entry: Admin or System
+  "version": // Version of the log entry, beginning with 0 and updated when a backfill or resend of the same log occurs
+  "log_id": // Unique ID of the log entry
+}
+```
+
+## Configure Audit Logging
+
+To set up Audit Logging, you must have an Amazon Web Services (AWS) account and set up Kinesis Data Streams.
+
+1. If you don't have an AWS account, follow the instructions from AWS in [Create and activate an AWS account](https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/).
+2. To set up Kinesis Data Streams, open the [AWS Management Console](https://aws.amazon.com/console/), search for Kinesis, and start the setup process.
+
+Be aware that Kinesis has a rate limit of 1,000 messages per second and quotas for both the number of records written and the size of the records.
+For more information, see [Why is my Kinesis data stream throttling?](https://aws.amazon.com/premiumsupport/knowledge-center/kinesis-data-stream-throttling/)
+
+### Create an Audit Log sink
+
+1. In the Temporal Cloud UI, select **Settings**.
+1. On the **Settings** page, select **Integrations**.
+1. In the **Audit Logging** card, select **Configure Audit Logs**.
+1. On the **Audit Logging** page, choose your **Access method** (either **Auto** or **Manual**).
+   - **Auto:** Configure the AWS CloudFormation stack in your AWS account from the Cloud UI.
+   - **Manual:** Use a generated AWS CloudFormation template to set up Kinesis manually.
+1. In **Kinesis ARN**, paste the Kinesis ARN from your AWS account.
+1. In **Role name**, provide a name for a new IAM Role.
+1. In **Select an AWS region**, select the appropriate region for your Kinesis stream.
+
+If you chose the Auto access method, select **Save and launch stack**.
+The Audit Log sink is configured and ready to use.
```

Comment:
After click Save and Launch stack, the user needs to enter a stack name in the next page and click "Create Stack" at the bottom right corner.  This page is also a AWS CloudFormation page, but is just a one click away from getting the whole thing set up.